scripts/README.md

```diff
@@ -6,40 +6,32 @@ A typical release goes like this:
 
 ### Prerequisites
 
-1. You must be a member of the `@mui` org in npm to publish the release
-2. Set up your npm authToken by logging into npm (`npm login`) . This will save a token to `~/.npmrc` as a line that looks
-   like this:
+1. You must be a member of the `@mui` org in npm to publish the release.
+2. Set up your npm authToken by logging into npm (`npm login`) . This will save a token to `~/.npmrc` as a line that looks like this:
    ```text
    //registry.npmjs.org/:_authToken=npm_000000000000000000000000000000000000
    ```
-3. Make sure you have added the `material-ui-docs` remote to deploy the documentation:
-   ```bash
-   git remote add material-ui-docs https://github.com/mui/material-ui-docs.git
-   ```
-4. Generate a GitHub Token at https://github.com/settings/personal-access-tokens/new and add it to your shell rc script (either `.bashrc` or `.zshrc`) as `GITHUB_TOKEN`.
+3. Generate a GitHub Token at https://github.com/settings/personal-access-tokens/new and add it to your shell rc script (either `.bashrc` or `.zshrc`) as `GITHUB_TOKEN`.
 
 ### Prepare
 
-The following steps must be proposed as a pull request.
-
 1. Generate the changelog with `pnpm release:changelog`
    The output must be prepended to the top level `CHANGELOG.md`
    `pnpm release:changelog --help` for more information. If your GitHub token is not in your env, pass it as `--githubToken <my-token>` to the above command.
 
-2. Clean the generated changelog:
-   1. Match the format of https://github.com/mui/material-ui/releases.
-   2. Change the packages names casing to be lowercase if applicable, for example change `Material` to `material`
-3. Update the root `/package.json`'s version
-4. Run `pnpm release:version`. Keep the package versions of stable public packages the same as the root `package.json` version.
-5. Open PR with changes and wait for review and green CI
-6. Merge PR once CI is green and it has been approved
+2. Clean the generated changelog matching the format of https://github.com/mui/base-ui/releases. Include only changes to the public packages (ignore docs, core, and similar PRs).
+3. Copy the clean changelog into `docs/src/app/(public)/(content)/react/overview/releases/page.mdx`. Remove the author handles from PRs (like "@michaldudak") and update links.
```

Comment:
This is currently cumbersome, but I plan to update the scripts to do it for us. When it's done, I'll update the instructions.